### PR TITLE
add interface to scikit regressors

### PR DESCRIPTION
**Pull request by felipeZ**
## New features
* Add interface to Scikit Regressor #85

### CODE REVIEW

File: swan/modeller/scikit_modeller.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        # Generate random indices to train and validate the model
+        size = len(self.fingerprints)
+        indices = np.arange(size)
+        np.random.shuffle(indices)
+
+        ntrain = int(size * frac[0])
+        self.features_trainset = self.fingerprints[indices[:ntrain]]
+        self.features_validset = self.fingerprints[indices[ntrain:]]
+        self.labels_trainset = self.labels[indices[:ntrain]]
+        self.labels_validset = self.labels[indices[ntrain:]]
```
Comment (NicoRenaud) [inline swan/modeller/scikit_modeller.py:58]: if we start having many modelers we might want to put all the boiler plate code into a base class ? I'm not sure it's really needed now already or not. Your call :)

**Review by NicoRenaud — APPROVED** [review]
that was fast :) Looks good. One small remark about potentially creating and modeller base class if that's worth the effort :) 👍

File: swan/modeller/scikit_modeller.py (hunk 1 above)
Comment (felipeZ) [reply swan/modeller/scikit_modeller.py:58]: I have considered introducing a base class for the modellers family, but there are almost not overlapping code between the two modellers that we have so far. Maybe we can introduce a base class in the future if we add interfaces to other ML packages. Do you agree?

Comment (NicoRenaud) [reply swan/modeller/scikit_modeller.py:58]: sounds good !

**Pull request merged**